src/Serilog.Sinks.BrowserHttp/LoggerConfigurationBrowserHttpExtensions.cs

```diff
@@ -81,5 +78,68 @@ public static class LoggerConfigurationBrowserHttpExtensions
 
             return loggerSinkConfiguration.Sink(sink, restrictedToMinimumLevel);
         }
+
+        /// <summary>
+        /// Adds a sink that writes log events to an HTTP server.
+        /// </summary>
+        /// <param name="loggerSinkConfiguration">The logger configuration.</param>
+        /// <param name="endpointUrl">The URL of the server logging endpoint.</param>
+        /// <param name="restrictedToMinimumLevel">The minimum log event level required 
+        /// in order to write an event to the sink.</param>
+        /// <param name="batchPostingLimit">The maximum number of events to post in a single batch.</param>
+        /// <param name="period">The time to wait between checking for event batches.</param>
+        /// <param name="eventBodyLimitBytes">The maximum size, in bytes, that the JSON representation of
+        /// an event may take before it is dropped rather than being sent to the server. Specify null for no limit.
+        /// The default is 265 KB.</param>
+        /// <param name="controlLevelSwitch">If provided, the switch will be updated based on the server's level setting
+        /// for the corresponding API key. Passing the same key to MinimumLevel.ControlledBy() will make the whole pipeline
+        /// dynamically controlled. Do not specify <paramref name="restrictedToMinimumLevel"/> with this setting.</param>
+        /// <param name="httpClient">The HttpClient that will send the log messages to the ingestion endpoint.</param>
+        /// <param name="queueSizeLimit">The maximum number of events that will be held in-memory while waiting to ship them to
+        /// the server. Beyond this limit, events will be dropped. The default is 100,000. Has no effect on
+        /// durable log shipping.</param>
+        /// <returns>Logger configuration, allowing configuration to continue.</returns>
+        /// <exception cref="ArgumentNullException">A required parameter is null.</exception>
+        public static LoggerConfiguration BrowserHttp(
+            this LoggerSinkConfiguration loggerSinkConfiguration,
+            HttpClient httpClient,
+            string endpointUrl = DefaultOriginEndpointUrl,
+            LogEventLevel restrictedToMinimumLevel = LevelAlias.Minimum,
+            int batchPostingLimit = BrowserHttpSink.DefaultBatchPostingLimit,
+            TimeSpan? period = null,
+            long? eventBodyLimitBytes = 256 * 1024,
+            LoggingLevelSwitch controlLevelSwitch = null,
+            int queueSizeLimit = BrowserHttpSink.DefaultQueueSizeLimit,
+            IDictionary<string, string> defaultRequestHeaders = null)
```

Comment:
There's probably no reason to accept these here, since a) if an `HttpClient` is passed in, then the caller can configure its default request headers, and b) it'd be strange to globally modify the passed in client this way.